utils/utils/src/main/scala/pl/touk/nussknacker/engine/util/SynchronousExecutionContextAndIORuntime.scala

```diff
@@ -0,0 +1,47 @@
+package pl.touk.nussknacker.engine.util
+
+import cats.effect.unsafe.{IORuntime, IORuntimeConfig, Scheduler}
+import com.typesafe.scalalogging.LazyLogging
+
+import java.time.Instant
+import java.time.temporal.ChronoField
+import java.util.concurrent.Executor
+import scala.concurrent.ExecutionContext
+import scala.concurrent.duration.FiniteDuration
+
+object SynchronousExecutionContextAndIORuntime extends LazyLogging {
+
+  implicit val syncEc: ExecutionContext = createSyncExecutionContext()
+  implicit val syncIoRuntime: IORuntime = createSyncIORuntime()
+
+  private def createSyncExecutionContext(): ExecutionContext = ExecutionContext.fromExecutor(new Executor {
+    def execute(task: Runnable): Unit = task.run()
+  })
+
+  private def createSyncIORuntime(): IORuntime = {
+    IORuntime(
+      compute = syncEc,
+      blocking = syncEc,
+      scheduler = DummyScheduler,
+      shutdown = () => {},
+      config = IORuntimeConfig()
+    )
+  }
+
+  private object DummyScheduler extends Scheduler {
+
+    override def sleep(delay: FiniteDuration, task: Runnable): Runnable = {
```

Comment:
Can we leave some comment why this scheduler won't be used? 
If I understand correctly, this is because we only use flatMap/map methods on the effect in the interpreter and no time dependent callbacks.